requirements/dev.txt

```diff
@@ -1,6 +1,6 @@
 #
-# This file is autogenerated by pip-compile with python 3.9
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
Thanks for reverting the requirements changes. It's great to have them regenerated with the recent updates from `kytos` transitive dependencies, however, to avoid confusion, could you regenerated with py `3.9` that's the supported version? The result will probably be the same, but to avoid surprises. Also if you could keep `typing-extensions` pinned as `typing-extensions>=4.0.1`

> Starting with version 4.0.0, typing_extensions uses [Semantic Versioning](https://semver.org/). The major version is incremented for all backwards-incompatible changes. Therefore, it's safe to depend on typing_extensions like this: typing_extensions >=x.y, <(x+1), where x.y is the first version that includes all features you need.